src/main/scala/decider/PathConditions.scala

```diff
@@ -221,7 +221,7 @@ private[decider] class LayeredPathConditionStack
        with PathConditionStack
        with Cloneable {
 
-  private var layers: Stack[PathConditionStackLayer] = Stack.empty
+  var layers: Stack[PathConditionStackLayer] = Stack.empty
```

Comment:
Please keep `/* private */` to indicate that the field should not be accessed, or at least not modified, from the outside.